Vagrantfile

```diff
@@ -298,6 +310,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                   ENV["CONTIV_CLUSTER_STORE"] || "etcd://localhost:2379",
                   ENV["CONTIV_DOCKER_VERSION"] || docker_version,
                   ENV['CONTIV_NODE_OS'] || "",
+                  ENV['CONTIV_V2PLUGIN_NAME'] || "contiv/v2netplugin:0.1",
```

Comment:
Just put the default name as contiv/netplugin:0.1, but we can override this with the env variable. Will update the release scripts to create/release plugin with release version tag as a new PR